public/app/features/alerting/unified/components/mute-timings/MuteTimingForm.tsx

```diff
@@ -96,15 +98,22 @@ const MuteTimingForm = ({ muteTiming, showError, provenance }: Props) => {
       },
     };
 
-    dispatch(
+    const saveAction = dispatch(
       updateAlertManagerConfigAction({
         newConfig,
         oldConfig: result,
         alertManagerSourceName: alertManagerSourceName!,
         successMessage: 'Mute timing saved',
         redirectPath: '/alerting/routes/',
+        redirectSearch: 'tab=mute_timings',
       })
     );
+
+    setUpdating(true);
```

Comment:
We're disabling the form when the AM is still updating with changes – previously we immediately redirected and hoped it would have saved the changes.